projects/components/CHANGELOG.MD

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 - Add translation for `Filter By` label in Quick search component
 
+## [2.0.0-dev.26]
+## Changed
+- Adjust the format of Catalog Sharing Modal data
```

Comment:
Please be more specific with this message

Change record format in UsersGroupsOrgsSharingModalDirective to be RECORDS